src/Tribe/Main.php

```diff
@@ -363,20 +363,20 @@ public function load_text_domain( $domain, $dir = false ) {
 		}
 
 		$locale = get_locale();
-		$mofile = WP_LANG_DIR . '/plugins/' . $domain . '-' . $locale . '.mo';
+		$plugin_rel_path = WP_LANG_DIR . '/plugins/';
```

Comment:
Are we sure not passing the correct file works?Or is it something specific to Windows?